### PR TITLE
Added support for sum and average on meta

### PR DESCRIPTION
**Pull request by marcopesani**
I needed to extend the meta function in order to support sum and average of a specific attribute. I hope it is useful :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +67,7 @@
         "graphql-tools": "~1.1.0",
         "graphql-type-json": "~0.1.4",
         "inflection": "~1.12.0",
+        "lodash": "^4.17.4",
```
Comment (djhi) [inline package.json]: Not used anywhere. Besides, if some lodash functions are needed, just import them as independent packages as we did for merge.

Comment (marcopesani) [reply package.json]: Removed

**Comment by djhi** [comment]
Sorry, can you rebase ?

**Comment by fzaninotto** [comment]
I must say I'm rather -1 for adding this to the library. I'm not sure it's common enough to be of use for any one else but you. Besides, the requirement to specify an `attr` (bad name by the way) complexifies the `getList` API imo.

**Pull request closed without merging**